style.css

```diff
@@ -387,4 +398,78 @@
     font-weight: 600;
     line-height: 24px;
   }
+
+  /* CONTACT SECTION STARTS HERE ------------------------------------------------ */
+
+  .forms-section {
+    background-color: #6070ff;
+    background-image: url(assets/mobile/contact\ background.png);
+    background-size: cover;
+    background-repeat: no-repeat;
+    border-top-left-radius: 70px;
+    position: relative;
+    top: -80px;
+  }
```

Comment:
- [ ] Your form style is great, but the `top: -80px;` attribute is creating whitespace at the bottom of the page. Kindly fix this issue and remove the whitespace to make your page looks more professional and similar to the Figmat template. (by default the `body` element has a margin on some browsers, it is nice to set the margin of the `body` element to 0 so the design looks consistent)
 - Your design:
 - 
<img width="436" alt="image" src="https://user-images.githubusercontent.com/61919617/207385500-64e231f7-647b-412f-87be-664de131ac2c.png">

Reply:
Fixed 😄 🚀